spec/student_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../book'
+require_relative '../student'
+require_relative '../rental'
+require 'json'
+
+describe Student do
+  before :each do
+    @student = Student.new 24, 'Eddie'
+  end
+  context 'When testing the Student class' do
+    it 'Given age and a name Object must be returned' do
+      expect(@student).to be_an_instance_of Student
+    end
+  end
```

Comment:
- [ ] Good job, but some tests are missing, you are missing tests to check if `The constructor method sets the default value for the optional name parameter if it is not provided `